python/paddle/fluid/tests/unittests/mkldnn/test_pool2d_mkldnn_op.py

```diff
@@ -160,4 +191,6 @@ def init_shape(self):
 
 
 if __name__ == '__main__':
+    from paddle import enable_static
+    enable_static()
```

Comment:
Why use `enable_static` only in this mkldnn unit-test?

Reply:
@luotao1 It should be in every test , it just I was working on that one so I added only in this one.

Reply:
Got it.